bundles/org.openhab.binding.matter/README.md

```diff
@@ -0,0 +1,472 @@
+
+**Please note this binding requires openHAB 4.3 builds or later that support discovery codes when searching for devices.** 
+
+## Supported functionality
+
+This binding supports 2 different types of matter functionality which operate independently of each other.
+
+- [Matter Client](#matter-client)
+  - This allows openHAB to discover and control other Matter devices like lights, thermostats, window coverings, locks, etc...
+
+- [Matter Bridge](#matter-bridge)
+  - This allows openHAB to expose items as Matter devices to other Matter clients.  This allows local control of openHAB devices from other ecosystems like Apple Home, Amazon, and Google Home.
+
+For more information on the Matter specification, see the [Matter Ecosystem Overview](#matter-ecosystem-overview) section at the end of this document.
+
+## Matter.JS Runtime
+
+This binding uses the excellent [matter.js](https://github.com/project-chip/matter.js) implementation of the the Matter 1.4 protocol.
+
+As such, this binding requires NodesJS 18+ and will attempt to download and cache an appropriate version when started if a version is not already installed on the system. Alpine Linux users (typically docker) and those on older Linux distributions will need to install this manually as the official NodeJS versions are not compatible.
+
+## Matter and IPv6
+
+Matter **requires** IPv6 to be enabled and be routable between openHAB and the Matter device.  This means IPv6 needs to be enabled on the host openHAB is running, and the network must be able route IPv6 unicast and multicast messages.  Docker, VLANs, subnets and other configurations can prohibit Matter from working if not configured correctly.
+
+
+# Matter Client
+
+This describes the Matter controller functionality for discovering and controlling Matter devices.
+
+## Supported Things
+
+The Matter Binding supports the following types of things:
+
+- `controller`: The main controller that interfaces with Matter devices. It requires the configuration parameter  `nodeId` which sets the local Matter node ID for this controller (must be unique in the fabric).  **This must be added manually.**
+- `node`: Represents an individual Node within the Matter network. The only configuration parameter is `nodeId`.  A standard Node will map Matter endpoints to openHAB channel groups. **This will be discovered automatically** when a pairing code is used to scan for a device and should not be added manually.
+- `endpoint`: Represents an standalone endpoint as a child of a `node` thing. Only Endpoints exposed by Matter bridges will be added as `endpoint` things, otherwise Matter Endpoints are mapped on a `node` thing as channel groups.  An `endpoint` thing **will be discovered automatically** when a node is added that has multiple bridged endpoints and should not be added manually.  
+
+**Note: The `bridged-endpoint` thing type is deprecated in favor of the `endpoint` thing type and will be removed in a future release.**
+
```

Comment:
I guess this is mostly targeting early adopters/testers of the community version of the binding. Do you think this should stay in the official documentation with 5.0?